### DISCUSSION TITLE
PatchTool 6.11 5/8

### DISCUSSION THREAD

**Discussion by GpointChen** [GpointChen/FFXIVChnTextPatch-GP#163] [Announcements]
- 加入中國服6.05（+6.08）翻譯
- 修正大量簡繁轉換與錯別字
- 修正旅行筆錄的錯誤

其他版本會晚點再上傳。

檔案：
- FFXIVChnTextPatch-0508.rar：本體。一般而言，下載這個即可。JRE縮小過。

選用：
- [第三方4K字體包](https://github.com/GpointChen/FFXIVChnTextPatch-GP/releases/download/v2.53/font-4k.rar)：部分解析度設定下會出現字體錯位等情形，請自行照需求使用。解壓後改名為font放到resource資料夾下（取代原資料夾）。
- [FFXIV-jar-0211.rar](https://github.com/GpointChen/FFXIVChnTextPatch-GP/releases/download/v2.56/FFXIV-jar-0211.rar)：如果你有遇到JNI error，請參考 #90 放到與exe同位置使用。
- [backup-0508.rar](https://github.com/GpointChen/FFXIVChnTextPatch-GP/releases/download/v2.59/backup-0415.rar)：乾淨的0a0000檔案。如果漢化出現問題，可以將這些檔案放到backup後按還原（或是手動取代遊戲檔案），然後再次嘗試漢化。
- [rawexd-ja-611-0508.rar](https://github.com/GpointChen/FFXIVChnTextPatch-GP/releases/download/v2.59/rawexd-ja-610-v2-0415.rar)：日文原文的CSV。

有問題歡迎至Issues或Discussions回報。

<hr /><em>This discussion was created from the release <a href='https://github.com/GpointChen/FFXIVChnTextPatch-GP/releases/tag/'>PatchTool 6.11 5/8</a>.</em>

**Comment by Logan8246EX** [comment]
感謝

**Comment by QOO5** [comment]
萬魔殿零式P4S 沒有顯示遠近

**Comment by Teimi** [comment]
感謝

**Comment by NezukoAnya** [comment]
为什么他说需要java runtime environment 16.0.1

**Comment by siler2011** [comment]
感謝大大!
